AgriParcel/schema.json

```diff
@@ -1,162 +1,180 @@
-        },
-        "belongsTo": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.3",
+    "modelTags": "",
+    "$id": "https://smart-data-models.github.io/dataModel.Agrifood/AgriParcel/schema.json",
+    "title": "Smart Data Models - Agri Parcel",
+    "description": "This entity contains a harmonised description of a generic parcel of land. This entity is primarily associated with the agricultural vertical and related IoT applications.",
+    "type": "object",
+    "allOf": [
+        {"$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"},
+        {"$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"},
+        {"$ref": "https://smart-data-models.github.io/dataModel.Agrifood/agrifood-schema.json#/definitions/AgriFood-Commons"},
+        {"properties": {
+            "type": {
+                "type": "string",
+                "enum": ["AgriParcel"],
+                "description": "Property. NGSI Entity Type. It has to be AgriParcel"
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
-            }
-          ],
-          "description": "Relationship. Entity the item belongs to"
-        },
-        "ownedBy": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+            "area": {
+                "type": "number",
+                "minimum": 0,
+                "description": "Property. Model:'http://schema.org/Number'. Units:'m2'. The area of the parcel nominally in square meters."
             },
-            {
-              "type": "string",
-              "format": "uri",
-              "description": "Property. Identifier format of any NGSI entity"
-            }
-          ],
-          "description": "Relationship. Owner (Person or Organization) of the item"
-        },
-        "hasAgriParcelParent": {
-          "anyOf": [
-            {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 256,
-              "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
-              "description": "Property. Identifier format of any NGSI entity"
+            "category": {
```

Comment:
should be an enumerated clause here? 
This was the case, please include it. 
Additionally, if these categories are coming from an existing regulation/standard please include in the description a link to it.

Reply:
Something must be gotten corrupted in my proposed changes. The only changes I was supposed to make were two additional elements, soilTextureType and irrigationSystemType. Not really sure why additional changes were proposed in the PR?

Reply:
I am commenting on this based on the description you provide. If we have a description indicating a group of values, with the clause Enum:'' it would coherent to have enum:[] in the json schema with these values.

Reply:
I´ve committed the changes we discussed in the latest version. Made some changes to enum values in soilTextureType and added references to where the enum values for soilTextureType and irrigationSystemType are defined.